### PR TITLE
Add unit tests for neg, neg_, tanh, and tanh_

### PR DESCRIPTION
**Pull request by joewie**
Resolves: #73

### CODE REVIEW

File: tests/test_math.py [hunk 1]
```diff
@@ -47,6 +47,20 @@ def test_floor(self):
...
+        # int
+        t1 = TensorBase(np.array([[-0, 1, -2], [0, -1, 2]]))
+        t2 = syft.math.tanh(t1)
+        self.assertTrue(np.array_equal(t1.data, np.array([[0, 1, -2], [0, -1, 2]])))
+        self.assertTrue(np.array_equal(t2.data, np.array([[0, np.tanh(1), np.tanh(-2)], [0, np.tanh(-1), np.tanh(2)]])))
+        # float
+        t1 = TensorBase(np.array([[-0.0, 1.5, -2.5], [0.0, -1.5, 2.5]]))
+        t2 = syft.math.tanh(t1)
+        self.assertTrue(np.array_equal(t1.data, np.array([[0.0, 1.5, -2.5], [0.0, -1.5, 2.5]])))
+        self.assertTrue(np.array_equal(t2.data, np.array([[0.0, np.tanh(1.5), np.tanh(-2.5)], [0.0, np.tanh(-1.5), np.tanh(2.5)]])))
```
Comment (bharathgs) [inline tests/test_math.py]: why not this ? : `np.tanh(np.array([[0, 1, -2], [0, -1, 2]]))`

Comment (bharathgs) [reply tests/test_math.py]: and `np.tanh(np.array([[0.0, 1.5, -2.5], [0.0, -1.5, 2.5]]))`

Comment (joewie) [reply tests/test_math.py]: Hmm I'm guessing that I was going for slightly less abstraction. I don't feel strongly about it, do you want me to change it?

Comment (bharathgs) [reply tests/test_math.py]: yes.

**Pull request merged**